spec/integration/mappers/deep_embedded_spec.rb

```diff
@@ -2,7 +2,7 @@
 require 'rom/adapter/memory'
 
 describe 'Mappers / deeply embedded tuples' do
-  let(:setup) { ROM.setup('memory://test') }
+  let(:setup) { ROM.setup(:memory, 'memory://test') }
```

Comment:
@solnic all of these will turn into `ROM.setup(:memory)` which is probably a sign that the change is a good thing.